sqlstore/src/main/resources/db/migration/V0035__add_description_to_pool.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE Pool ADD COLUMN description varchar(255) DEFAULT 'Pool';
```

Comment:
Nope, must be vestigial from when I was considering making it not nullable. I'll update that.